### PR TITLE
doc: remove plain pip installation instructions

### PR DESCRIPTION
**Pull request by nodiscc**
only support virtualenv to prevent python dependency/path issues

### CODE REVIEW

File: docs/user/installation.rst [hunk 1]
```diff
@@ -40,7 +34,7 @@ Here is an example using a Python 3.5 interpreter:
   requests-jwt==0.4
   shaarli-client==0.1.0
 
-Get the source code
+From the source code
```
Comment (virtualtam) [inline docs/user/installation.rst]: there must be as many underlining dashes as there are characters in a title (yup, this is one of the few annoying sides of rst)

**Comment by ArthurHoaro** [comment]
If this client is available through Python's package manager, shouldn't it be documented?

**Comment by virtualtam** [comment]
@ArthurHoaro 3 installation procedures are currently documented:
1. install system-wide from PyPI with pip
2. create a local (user) virtualenv and install from PyPI with pip
3. create a local (user) virtualenv and install from sources

Option 2. is the recommended approach and should fit most use cases.

This PR removes documentation for option 1., which is generally considered bad practice, as packages installed this way are likely to overwrite/conflict with resources otherwise handled by the distro's package manager.

It is exceptionally used in the following situations:
- the package is not available through the distro's repositories, nor commonly used third-party repositories (e.g. PPAs)
- when building a Docker image that is dedicated to running a specific set of packages

**Comment by ArthurHoaro** [comment]
Thanks for the explanation, it makes sense.

**Comment by nodiscc** [comment]
Besides, to avoid running pip as root, one has to use `pip --user install`, but then the proper [`$PYTHONPATH`](https://docs.python.org/2/using/cmdline.html#envvar-PYTHONPATH) environment variable must be set, else python would not know where to import modules from. This can be confusing - `virtualenv` takes care of all these intricacies, hence it is better to only support installation in a virtualenv which is guaranteed to work out of the box.

**Pull request merged**